client/assets/js/board.js

```diff
@@ -3,49 +3,57 @@
 import createCanvasGame from './createCanvasGame.js';
 import Snake from './snake';
 import Apple from './apple';
+import $ from 'jquery';
 
-const DELAY = 500;
+const DELAY = 150;
 
 export default class Board {
 
 // A snake store his position (x and y) and his score
 	constructor() {
-		this.snakes = [];
 		this.context = createCanvasGame();
+		this.snakes = [];
 		this.apples = [];
 	}
 
-	newSnake(x, y){
+	newSnake(x, y) {
+		let snake = new Snake(this.context, x, y);
+		snake.draw();
 
+		// Temp : Simulate 4 bodyParts on the snake
+		snake.addBodyPart(x - 60, y);
```

Comment:
These are just temporary numbers, like those in main.js when creating a snake are too.